bankapp/src/main/kotlin/com/clouway/bankapp/adapter/gae/pubsub/AsyncTransactionListener.kt

```diff
@@ -0,0 +1,25 @@
+package com.clouway.bankapp.adapter.gae.pubsub
+
+import com.clouway.pubsub.core.EventBus
+import com.clouway.pubsub.core.event.DepositMadeEvent
+import com.clouway.pubsub.core.event.WithdrawMadeEvent
+
+/**
+ * @author Tsvetozar Bonev (tsbonev@gmail.com)
+ */
+class AsyncTransactionListener(private val eventBus: EventBus) : TransactionListener {
```

Comment:
AsyncTransactionListener -  this name means that you have asyncronous listener of the tranasactions, but you actually have syncronous implementation of the transations events and you fire them as an asyncronoys (pub-sub) evetns.
suggest TransactionAsyncEventPublisher(Notifier)

Reply:
Implemented Announcer and changed listener class names to TransactionAsyncEventPublisher and UserChangeAsyncEventPublisher.